release-notes.md

```diff
@@ -1,4 +1,43 @@
 # Rigel-18.19.0
+## Support user id hashing ##
+
+* Issue Type: Task
+* Issue ID: - Issue Type: Feature
+- Issue ID: PLAT-24004
+
+#### Deployment ####
+Execute the curl command, replace esearch_host, esearch_port and kaltura_kuser index (default it 'kaltura_kuser')
+##### Note: command below is for elastic 7.x.x version, if you have different version, please refer to elastic documentations on how to update index mapping #####
+Elastic docs: https://www.elastic.co/guide/en/elasticsearch/reference/7.10/indices-put-mapping.html
+
+{ "external_id": { "type": "keyword", "normalizer": "kaltura_keyword_normalizer" }, "is_hashed": { "type": "boolean"} }
```

Comment:
you put in the example external_id and is_hashed but on the curl you put the 4 new types
is line 13 even needed?